data-prepper-plugins/mutate-event/build.gradle

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Let's keep the naming for the folder structure consistent. 

`data-prepper-plugins/mutate-event-processors/`